src/components/presentation/shareButton/Share.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FacebookShareButton,
+  FacebookIcon,
+  TwitterShareButton,
+  TwitterIcon,
+  GooglePlusShareButton,
+  GooglePlusIcon,
+} from 'react-share';
+
+class Share extends React.Component {
+  state = {
+    toggleShare: false,
+  }
+
+  handleClick = () => {
```

Comment:
@frostyblok Awesome work. Quick one... event handler names should be descriptive enough for someone reading your code without having to look at where it's been used.

`handleClick` is kind of ambiguous don't you think? Handle click for what? A text, link, form?

I think this should be refactored to `onShareClick`, `onShareIconClick` or `handleShareIconClick`

Reply:
Good spot Chief! I will work on that ASAP!